apps/omg_eth/lib/eth/root_chain.ex

```diff
@@ -228,6 +248,35 @@ defmodule OMG.Eth.RootChain do
          do: {:ok, Enum.map(logs, &decode_exit_started/1)}
   end
 
+  defp get_inflight_data(hash) do
+    {:ok, eth_tx} = Ethereumex.HttpClient.eth_get_transaction_by_hash(hash)
+    <<"0x", encode_inflight::binary>> = eth_tx["input"]
```

Comment:
instead of this pattern match and later `Base.decode` use `Eth.Encoding.from_hex()`